### PR TITLE
Remove blog tags and set up redirect

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/KZX8oBwO/6898

### CODE REVIEW

File: config/redirects.yml [hunk 1]
```diff
@@ -395,6 +396,17 @@ redirects:
   "/blog/tag/career-change-to-teaching": "/life-as-a-teacher/teaching-as-a-career/change-careers"
   "/blog/tag/changing-career-to-teaching": "/life-as-a-teacher/teaching-as-a-career/change-careers"
   "/blog/tag/interviews": "/how-to-apply-for-teacher-training/teacher-training-interview"
+  "/blog/tag/applications": "/how-to-apply-for-teacher-training/teacher-training-application"
```
Comment (gemmadallmandfe) [inline config/redirects.yml:399]: would be good to order **all** of these tag redirects either alphabetically or maybe by destination?

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for this to be merged after a tech review

**Comment by gemmadallmandfe** [comment]
@martyn-w I noticed that you had to re-add `app/webpacker/images/content/blog/notepad.jpg` as we are using it in the How to apply section (ie outside the blog)

We have a tech debt ticket on removing all references to the blog from the code so I will make a note on the ticket that we need to rename this image at the same time, to avoid confusion!

**Review by martyn-w — APPROVED** [review]
👍  LGTM

**Pull request merged**